data/locale/en-US.ini

```diff
@@ -140,6 +140,7 @@ AdvSceneSwitcher.macroTab.highlightPerformedActions="Highlight recently performe
 AdvSceneSwitcher.macroTab.newMacroRegisterHotkey="Register hotkeys to control the pause state of new macros"
 AdvSceneSwitcher.macroTab.currentDisableHotkeys="Register hotkeys to control the pause state of selected macro"
 AdvSceneSwitcher.macroTab.currentSkipExecutionOnStartup="Skip execution of actions of current macro on startup"
+AdvSceneSwitcher.macroTab.currentStopActionsIfNotDone="Stop and rerun actions if macro meets its conditions again before it finishes running, instead of skipping reexecution"
```

Comment:
How about changing this to:
`Stop the actions of this macro, if they are running still, when a new execution is triggered.`

The conditions won't necessarily always be involved as the macro can also be started manually or via the "Macro" action.

Reply:
Good point, should include rerun info though.